cogs/images.py

```diff
@@ -71,11 +74,16 @@ async def on_ready(self):
             os.mkdir("./tmp/", mode=0o755)
 
     @staticmethod
-    async def get_attachment(ctx: discord.ext.commands.Context):
-        messages = await ctx.channel.history(limit=Images.IMAGE_HISTORY_LIMIT
-                                             ).flatten()
-        return next(
-            (msg.attachments[0] for msg in messages if msg.attachments), None)
+    async def get_attachment(ctx):
```

Comment:
not that we're particularly consistent on this but why is the type hint removed here?